app/models/tasks/copy_image_task.rb

```diff
@@ -12,9 +12,21 @@ def start!
   def send_rexif_job
     return if !image_resource.original_url
 
-    parts = image_resource.original_url.gsub(/^s3:\/\//, '').split('/', 2)
-    source_bucket_name = parts[0]
-    source_object_key = parts[1]
+    if image_resource.original_url.match(/^s3:/)
```

Comment:
Would be nicer to just move this to a separate method, and return nil in the `else` (and maybe `logger.warn()` it?)

Then

```ruby
if rexif_source && ENV['REXIF_JOB_EXECUTION_SNS_TOPIC']
```